cat/CaT-service.yaml

```diff
@@ -5085,6 +5085,12 @@ components:
                 date:
                   readOnly: true
                 author:
+                  type: object
+                  properties:
+                    id:
```

Comment:
This should already be in place within [https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/compon[…]Enquiry](https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/Enquiry)
[15:21](https://ccs-scale.slack.com/archives/D02TS0QQZCP/p1646925707797569)

This is only need to support the CCS compiler being used